src/main/kotlin/ar/edu/unahur/obj2/vendedores/Centros.kt

```diff
@@ -0,0 +1,27 @@
+package ar.edu.unahur.obj2.vendedores
+
+import java.lang.Exception
+
+class Centro(ciudad: Ciudad) {
+    val vendedores = mutableListOf<Vendedor>()
+
+    fun addVendedor(vendedor: Vendedor) {
+        if (this.vendedores.contains(vendedor)) {
+            throw Exception("Este vendedor ya trabaja en el centro de distribucion")
+        }
+        else {
+            this.vendedores.add(vendedor)
+        }
+    }
```

Comment:
Si el flujo pasa por la rama del error la ejecución se corta, por lo que el `else` está de más.

```suggestion
    fun addVendedor(vendedor: Vendedor) {
        if (this.vendedores.contains(vendedor)) {
            throw Exception("Este vendedor ya trabaja en el centro de distribucion")
        }
        this.vendedores.add(vendedor)
    }
```